docs_source/conf.py

```diff
@@ -46,22 +46,29 @@
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-html_theme = "renku"
+html_theme = "sphinx_book_theme"
```

Comment:
you need to add this theme to the requirements package for the developer version.